packaging/ubuntu-14.04/tests/integrationtests

```diff
@@ -23,6 +23,14 @@ sed -i 's/\(PermitRootLogin\|PasswordAuthentication\)\>.*/\1 yes/' /etc/ssh/sshd
 
 service ssh restart
 
+# Map snapd deb package pockets to core snap channels. This is intended to cope
+# with the autopkgtest execution when testing packages from the different pockets
+if [ -n "$(apt list snapd | grep \\-proposed)" ]; then
```

Comment:
Thanks @chipaca, I've pushed changes for branching with the grep exit status, good point. If apt output doesn't match proposed nor updates then the default channel would be used, edge (this is the expected behavior when executing autopkgtest during PRs, the other channels will be used for SRUs).

About the duplication, this code is meant to be run during autopkgtest, we should need to find the path from which to source the file with the common snippet, does this look sane?:
```
SCRIPT=$(readlink -f "$0")
SCRIPT_PATH=$(dirname "$SCRIPT")
source "${SCRIPT_PATH}/../../tests/lib/snippet.sh"
```